src/panels/lovelace/cards/hui-entity-button-card.js

```diff
@@ -0,0 +1,161 @@
+import { html } from "@polymer/polymer/lib/utils/html-tag.js";
+import { PolymerElement } from "@polymer/polymer/polymer-element.js";
+
+import "../../../components/ha-card.js";
+
+import toggleEntity from "../common/entity/toggle-entity.js";
+import stateIcon from "../../../common/entity/state_icon.js";
+import computeStateDomain from "../../../common/entity/compute_state_domain.js";
+import EventsMixin from "../../../mixins/events-mixin.js";
```

Comment:
We should no longer use the EventsMixin, use fireEvent instead. See #1772

Reply:
Receiving Node.Dispatch is not a function when trying to use FireEvent

Reply:
what do you pass in as node?

Reply:
You need to do `fireEvent(this, <other args>)` instead of `this.fire(<other args>)`